src/main/siteConfigurationHandler.js

```diff
@@ -46,55 +46,60 @@ fluid.defaults("gpii.app.siteConfigurationHandler", {
                     ]
                 }
             },
-            target: "{app qssWrapper}.options.settingOptions.hiddenSettings"
+            target: "{that qssWrapper}.options.settingOptions.hiddenSettings"
```

Comment:
Why did all of these get changed - do you remember?

Reply:
This was changed by me at fixing the issue Javi had with overriding `{defaultSettingsLoader}.options.defaultSettingsUrl` using the url defined in the site config file. In [this fix](https://github.com/javihernandez/gpii-app/pull/4/files), `gpii.app.siteConfigurationHandler` was [moved to the top level](https://github.com/GPII/gpii-app/pull/179/files#diff-bc658f47466ba31df21972f46d3f02d4R641) where `gpii.appWrapper` is and `app` is now a sub-component of it. So I changed target from `app` to `that`.